tests/test_data_loader.py

```diff
@@ -114,3 +115,11 @@ def test_gen_crop_outside_boxes(self):
         instance = {"bbox": [10, 10, 100, 100], "bbox_mode": BoxMode.XYXY_ABS}
         with self.assertRaises(AssertionError):
             detection_utils.gen_crop_transform_with_instance((10, 10), (15, 15), instance)
+
+    def test_imagelist_padding_shape(self):
+        class TensorToImageList(torch.nn.Module):
+            def forward(self, tensor: torch.Tensor):
+                return ImageList.from_tensors([torch.ones((3, 10, 10))], 4).tensor
+
+        func = torch.jit.trace(TensorToImageList(), torch.ones((3, 10, 10), dtype=torch.float32))
+        func(torch.ones((3, 15, 20), dtype=torch.float32))
```

Comment:
could you also test the changes on the `batch_shape` branch?

Reply:
Sure